[Community] Update the Percona XtraDB Cluster Operator

tduzan opened the pull request:
There was an entry in Operator Hub for Percona's community operator for Percona XtraDB Cluster that was created by a community member, however it linked to older container images and referenced the current release version (0.3.0).  As part of our journey towards OpenShift certification we've regenerated our operator metadata and aligned our community and certified (pending) operator metadata.

This changeset updates a few key items for us:
* Operator version is 0.3.0 and links to 0.3.0 container images
* Updated description matches how we describe our operators in our marketing material
* Operator path name is now percona-xtradb-cluster
* Operator package name is now percona-xtradb-cluster-operator

We have a second community operator for Percona Server for MongoDB which we will contribute soon as well, and we should also later contribute certified versions of both operators to Operator Hub.  

Please review and let me know if any other changes are required.

SamiSousa commented on community-operators/percona-xtradb-cluster/perconaxtradbbackups.crd.yaml:
```diff
@@ -33,4 +37,4 @@ spec:
     - name: Age
       type: date
-      JSONPath: .metadata.creationTimestamp
+      JSONPath: .metadata.creationTimestamp
```
nit: Please add a newline

SamiSousa commented on community-operators/percona-xtradb-cluster/percona-xtradb-cluster.package.yaml:
```diff
@@ -0,0 +1,5 @@
+channels:
+    - name: alpha
+      currentCSV: percona-xtradb-cluster-operator.v0.3.0
+defaultChannel: alpha
```
nit: Please add a newline

tduzan commented:
Howdy @SamiSousa @robszumski @awgreene:

We have updated the PXC Operator from version 0.3.0 to version 1.0.0 which is our initial public GA release. We have now achieved certification for OpenShift for version 1.0.0 of the PXC Operator, and I have updated the contents of the Operator Hub manifest for community to match what was certified with our community images.  Please re-review and accept this PR if everything is in order.

Thanks!

tduzan commented:
I note the PR sanity check is failing because we have renamed the directory where this Operator lives.  This is due to the fact that "Percona" is the name of our company, the product name is Percona XtraDB Cluster.  We have also produced an operator for another product of ours, Percona Server for MongoDB, which will be submitted at a future date in a separate PR.  

Just some clarification as to why this PR touches two directories in apparent violation of the automated sanity checks.

Thanks

dmesser commented on community-operators/percona-xtradb-cluster/percona-xtradb-cluster-operator.v1.0.0.clusterserviceversion.yaml:
```diff
@@ -0,0 +1,238 @@
+  maturity: alpha
+  provider:
+    name: Percona
+  version: 1.0.0
```
This should also have a `replaces` field to denote that it replaces the v0.3.1 version.

dmesser commented:
/lgtm - directory change approved after discussion
